design/control-data-plane-separation/design.md

```diff
@@ -767,4 +776,17 @@ PASS
 ok  	command-line-arguments	17.727s
 ```
 
+### Performance goals
+
+- NKG can handle frequent configuration changes (1 change per second)
```

Comment:
Since the control plane batches events before  regenerating config and can also control how frequently to update the agents
```suggestion
- Agent can handle frequent configuration changes (1 change per second)
```